services/core/java/com/android/server/biometrics/sensors/fingerprint/aidl/FingerprintProvider.java

```diff
@@ -137,6 +137,8 @@ public class FingerprintProvider implements IBinder.DeathRecipient, ServiceProvi
     @Nullable private IVirtualHal mVhal;
     @Nullable private String mHalInstanceNameCurrent;
 
+    private boolean mCleanup;
+
```

Comment:
This var should be final

```suggestion
    private final boolean mCleanup;

```

Reply:
done

Reply:
thanks